src/shogun/evaluation/MachineEvaluation.h

```diff
@@ -8,24 +8,18 @@
 #ifndef CMACHINEEVALUATION_H_
 #define CMACHINEEVALUATION_H_
 
-#include <shogun/lib/config.h>
-
-#include <shogun/base/SGObject.h>
 #include <shogun/evaluation/Evaluation.h>
 #include <shogun/evaluation/EvaluationResult.h>
 #include <shogun/evaluation/MachineEvaluation.h>
+#include <shogun/lib/StoppableSGObject.h>
+#include <shogun/lib/config.h>
 
+#include <atomic>
```

Comment:
Do we need these headers (atomic, condition_variable, mutex)? I think we could delete them :)

Reply:
The build won't fail without those headers because we already have included them in previous headers file, so could you please remove them? :)